### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by RebelP72**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 
 <a href="https://discord.gg/PbBt4PQhpe"><img src="https://discordapp.com/api/guilds/865321790347018241/widget.png?style=banner3" alt="Discord Banner" align="right"/></a>
 
-**Create Robotics** is a mod about Robots and Robotics Automation in Minecraft using the Create mod.
+**Create Robotics** is a create addon that adds robotics and their automation to Minecraft;
```
Comment (OutCraft-Mods) [inline README.md]: Simple change, replace the `;` with a `.`
```suggestion
**Create Robotics** is a create addon that adds robotics and their automation to Minecraft. 
```

File: README.md [hunk 2]
```diff
@@ -7,6 +7,14 @@
 
 <a href="https://discord.gg/PbBt4PQhpe"><img src="https://discordapp.com/api/guilds/865321790347018241/widget.png?style=banner3" alt="Discord Banner" align="right"/></a>
 
-**Create Robotics** is a mod about Robots and Robotics Automation in Minecraft using the Create mod.
+**Create Robotics** is a create addon that adds robotics and their automation to Minecraft; 
+
+players can program the robots with {Placeholder name}, designed to be user-friendly, Simple, But still powerful.
```
Comment (OutCraft-Mods) [inline README.md]:
```suggestion
Players can program the robots with our own scripting-language called RoboScript, designed to be user-friendly, simple, but still powerful.
```

File: README.md [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
 <a href="https://discord.gg/PbBt4PQhpe"><img src="https://discordapp.com/api/guilds/865321790347018241/widget.png?style=banner3" alt="Discord Banner" align="right"/></a>
 
-**Create Robotics** is a mod about Robots and Robotics Automation in Minecraft using the Create mod.
+**Create Robotics** is a create addon that adds robotics and their automation to Minecraft; 
+
+players can program the robots with {Placeholder name}, designed to be user-friendly, Simple, But still powerful. 
+
+This allows the player to do many things with the robots, whether automating moving items from A to B, building a quarry, or simply using them to automate a mob grinder. 
+
+The mod adds almost infinite ways to automate with the robots."
```
Comment (OutCraft-Mods) [inline README.md]: Why a `"` at the end?
```suggestion
The mod adds almost infinite ways to automate with the robots!
```

File: README.md [hunk 4]
```diff
@@ -7,6 +7,14 @@
...
+**Create Robotics** is a create addon that adds robotics and their automation to Minecraft; 
+
+players can program the robots with {Placeholder name}, designed to be user-friendly, Simple, But still powerful. 
+
+This allows the player to do many things with the robots, whether automating moving items from A to B, building a quarry, or simply using them to automate a mob grinder. 
+
+The mod adds almost infinite ways to automate with the robots." 
+
+
+The syntax for {Placeholder name} can be found <a href="https://github.com/Worker20/CreateRobotics/wiki/Programming-Mechanic">here.</a>
```
Comment (OutCraft-Mods) [inline README.md]:
```suggestion
For help with RoboScript you can look <a href="https://github.com/Worker20/CreateRobotics/wiki/Programming-Mechanic">here</a>.
```

**Review by OutCraft-Mods — CHANGES_REQUESTED** [review]
@RebelP72 made some changes, could you please look into them?

**Comment by RebelP72** [comment]
@OutCraft-Mods Changes look good

**Comment by OutCraft-Mods** [comment]
Great! Thanks for the Readme update!

**Pull request merged**